### PR TITLE
Improve general user experience

### PR DESCRIPTION
**Pull request by manuelsanchez2**
In this pull request, I wanted to fix some small things:

- [X] Added useLocation in the footer - the active page shows a different opacity in the icon
- [X] Improve the favbutton - it does not work yet, but can be clicked
- [X] Link all the icons in the social area of Home.js
- [X] Link the profile icon to Profile.js
- [X] Link the doiche living logo to Home.js
- [X] Replace the facebook icon

### CODE REVIEW

File: client/src/components/FavButton/FavButton.js [hunk 1]
```diff
@@ -1,28 +1,45 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "@emotion/styled";
-import favoriteIcon from "../../assets/icons/favorite-icon.svg";
 import PropTypes from "prop-types";
 
 const Button = styled.button`
-  border: 1px solid var(--color-dark-border);
-  background: var(--color-transparent);
+  border: 0.1px solid var(--color-dark-border);
```
Comment (manuelpapa) [inline client/src/components/FavButton/FavButton.js]: Wow, I'm pretty sure, MQ screens won't be able to show this border. I would choose at least 0.5px or so.

File: client/src/components/FavButton/FavButton.js [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+      value ? "var(--color-primary)" : "var(--color-light-border)"};
   }
 `;
 
 const FavButton = ({ onClick, src, alt }) => {
+  const [checked, setChecked] = useState(false);
+  function handleChange() {
+    setChecked(!checked);
+  }
   return (
```
Comment (manuelpapa) [inline client/src/components/FavButton/FavButton.js:27]: Such a good way! Very great!

File: client/src/components/Footer/Footer.js [hunk 3]
```diff
@@ -24,20 +17,47 @@ const FooterNav = styled.nav`
   border-top: 1.5px solid var(--color-dark-border);
 `;
 
+const ImageFooter = styled.img`
+  width: 2rem;
+  height: 2rem;
+  margin: 12px 10px;
+  padding-bottom: 2.5px;
+  opacity: ${({ active }) => (active ? "1" : "0.5")};
```
Comment (manuelpapa) [inline client/src/components/Footer/Footer.js:25]: Now this is cool, I will do this too!

File: client/src/components/HomeSubscribeSectionField/HomeSubscribeSectionField.js [hunk 4]
```diff
@@ -16,13 +16,12 @@ const HomeSubscribeSectionField = styled.div`
...
     background: var(--color-gradient-submit);
-    margin: 0 auto;
-    margin-top: 6px;
+    margin: 1rem auto 0.7rem auto;
     padding: 0.5rem 2rem;
     color: var(--color-white);
     font-family: var(--font-lato);
     font-size: 1rem;
-    letter-spacing: 3px;
+    letter-spacing: 2px;
```
Comment (manuelpapa) [inline client/src/components/HomeSubscribeSectionField/HomeSubscribeSectionField.js]: You mix up `rem` and `px` values. Maybe you can decide for one 😃

File: client/src/components/ReturnButton/ReturnButton.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 
 const Button = styled.button`
   background: var(--color-white);
-  border: 1px solid var(--color-dark-border);
+  border: 0.1px solid var(--color-dark-border);
```
Comment (manuelpapa) [inline client/src/components/ReturnButton/ReturnButton.js]: Look the first comment. It's too small, bro.

Comment (manuelpapa) [reply client/src/components/ReturnButton/ReturnButton.js]: *That's what she said

**Review by manuelpapa — COMMENTED** [review]
Great app, Manu, I love the design and also the intuitive UX.

File: client/src/components/ReturnButton/ReturnButton.js (hunk 5 above)
Comment (manuelsanchez2) [reply client/src/components/ReturnButton/ReturnButton.js]: hahah thanks! you are right!

File: client/src/components/FavButton/FavButton.js (hunk 1 above)
Comment (manuelsanchez2) [reply client/src/components/FavButton/FavButton.js]: I wanted to get rid of it and I am still thinking about it!

File: client/src/components/FavButton/FavButton.js (hunk 2 above)
Comment (manuelsanchez2) [reply client/src/components/FavButton/FavButton.js:27]: Thanks!

File: client/src/components/HomeSubscribeSectionField/HomeSubscribeSectionField.js (hunk 4 above)
Comment (manuelsanchez2) [reply client/src/components/HomeSubscribeSectionField/HomeSubscribeSectionField.js]: You are right!

File: client/src/components/Footer/Footer.js (hunk 3 above)
Comment (manuelsanchez2) [reply client/src/components/Footer/Footer.js:25]: Try! it is cool yeah!

**Pull request merged**